app/forms/mappings/Formatters.scala

```diff
@@ -91,4 +91,26 @@ trait Formatters {
       override def unbind(key: String, value: A): Map[String, String] =
         baseFormatter.unbind(key, value.toString)
     }
+
+  private[mappings] def vatFormatter(invalidKey: String, maxLengthKey: String): Formatter[String] = new Formatter[String] {
+
+    val regexVatNumber = "(GB)?[0-9]+"
```

Comment:
Is the vat number a set length?

at the moment it will be valid as long as it's not _greater_ than 9 characters long. Therefore it could be shorter than 9, is that correct?